### PR TITLE
Feature: create `getStates` method

### PR DESCRIPTION
**Pull request by nolleto**
This method will return all Brazilian states sorted by name

#82

### CODE REVIEW

File: src/utilities/index.ts [hunk 1]
```diff
@@ -7,3 +7,4 @@ export { format as formatCEP, isValid as isValidCEP } from './cep';
 export { format as formatBoleto, isValid as isValidBoleto } from './boleto';
 export { format as formatCPF, generate as generateCPF, isValid as isValidCPF } from './cpf';
 export { format as formatCNPJ, generate as generateCNPJ, isValid as isValidCNPJ } from './cnpj';
+export * from './estados';
```
Comment (hyanmandian) [inline src/utilities/index.ts:10]: Can you replace `estados` by `state`?

Comment (nolleto) [reply src/utilities/index.ts:10]: Sure 😄

**Comment by hyanmandian** [comment]
Hey @nolleto , can you add some docs about this function?

**Comment by hyanmandian** [comment]
Add @nolleto as a contributor for code, doc, test.

**Comment by hyanmandian** [comment]
Closes #82

**Comment by hyanmandian** [comment]
@all-contributors please add @nolleto  for code, doc, and tests

**Pull request merged**